ci-operator/config/openshift/release/openshift-release-master__nightly-4.5.yaml

```diff
@@ -49,6 +49,7 @@ tests:
     pre:
     - ref: baremetalds-packet-setup
     - ref: ipi-install-rbac
+    - ref: baremetalds-devscripts-proxy
```

Comment:
I guess it could be the right occasion to use the recently introduced specialized workflows, ie `baremetalds-e2e-ipv4`?  See https://github.com/openshift/release/blob/32420fe0fb6bcdf14d4f616a0dd904e3835eb075/ci-operator/step-registry/baremetalds/e2e/ipv4/baremetalds-e2e-ipv4-workflow.yaml#L2

Reply:
done